app/Filament/Resources/EmailTemplateResource/Pages/ListEmailTemplates.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Filament\Resources\EmailTemplateResource\Pages;
+
+use Filament\Tables\Table;
+use App\Models\EmailTemplate;
+use Filament\Actions\CreateAction;
+use Assist\Division\Models\Division;
+use Filament\Tables\Actions\EditAction;
+use Filament\Resources\Pages\ListRecords;
+use Filament\Tables\Actions\BulkActionGroup;
+use Filament\Tables\Actions\DeleteBulkAction;
+use App\Filament\Columns\OpenSearch\TextColumn;
+use App\Filament\Resources\EmailTemplateResource;
+use Assist\Division\Filament\Resources\DivisionResource;
+
+class ListEmailTemplates extends ListRecords
+{
+    protected static string $resource = EmailTemplateResource::class;
+
+    public function table(Table $table): Table
+    {
+        return $table
+            ->columns([
+                TextColumn::make('name'),
+                TextColumn::make('model.name')
+                    ->label('Related To')
+                    ->getStateUsing(fn (EmailTemplate $record): ?string => $record->model->name)
```

Comment:
You are probably going to want to do the same thing you are doing for `url` there is no guarantee that all future models will have `name`